tos/tests/test_middleware.py

```diff
@@ -82,3 +84,70 @@ def test_accept_after_middleware_redirects_properly(self):
         # Confirm redirects back to the index page
         self.assertEqual(response.status_code, 302)
         self.assertEqual(response.url.replace('http://testserver', ''), str(reverse('index')))
+
+
+@override_settings(
+    MIDDLEWARE_CLASSES=settings.MIDDLEWARE_CLASSES + [
+        'tos.middleware.UserAgreementMiddleware',
+    ]
+)
+class BumpCoverage(TestCase):
```

Comment:
No worries, they're fine. I may push a more meaningful name for the class to master. :D